docs/src/less/pages/components/switches.less

```diff
@@ -0,0 +1,19 @@
+.switches-examples {
+  .clearfix();
+
+  .switches-example-container {
+    text-align: left;
+    margin-bottom: 16px;
+  }
+
+  .switches-example-group {
+    float: left;
+    width: 50%;
+  }
+
+  @media @device-medium {
+    .switches-example-group {
+      width: 33%;
```

Comment:
This file is a clone of buttons.less with this line being the only modification. I made this file so that pages/switches had its own css as opposed to sharing buttons.less with pages/button in case we ever wanted to make style changes to one page and not the other.